articles/resources/relnotes.md

```diff
@@ -16,6 +16,21 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version 0.xx.xxxx.xxx
+
+*Release date: July 1st, 2020*
+
+This release contains the following:
+
+- New qdk-chem chemistry format conversion tool 
```

Comment:
I forget the exact uid, but perhaps a link like the following would be helpful? Perhaps it would be good to have a brief page on how to install and use `qdk-chem` as well? Thoughts, @KittyYeungQ?

```suggestion
- New `qdk-chem` tool for converting legacy electronic structure problem serialization formats (e.g.: FCIDUMP) to [Broombridge](xref:todo)
```